src/openklant/components/klantinteracties/tests/test_admin.py

```diff
@@ -132,6 +135,57 @@ def test_digitaal_adres_inline(self):
         self.assertEqual(adres.adres, "email@example.com")
         self.assertIsNone(adres.betrokkene)
 
+    def test_voorkeurs_digitaal_adres_valid(self):
+        partij = PartijFactory.create()
+        digitaal_adres = DigitaalAdresFactory.create(partij=partij)
+        partij.voorkeurs_digitaal_adres = digitaal_adres
+
+        partij.clean()
+
+    def test_voorkeurs_digitaal_adres_invalid(self):
+        partij = PartijFactory.create(soort_partij=SoortPartij.persoon)
+        invalid_digitaal_adres = DigitaalAdresFactory.create()
+
+        form = PartijAdminForm(
+            data={
+                "voorkeurs_digitaal_adres": invalid_digitaal_adres.id,
+                "soort_partij": partij.soort_partij,
+            },
```

Comment:
Could you change it to receive only the error for the specific test case, passing all the required fields, that way you won't get error

```suggestion
            data={
                "uuid": uuid4(),
                "voorkeurs_digitaal_adres": invalid_digitaal_adres.id,
                "soort_partij": partij.soort_partij,
            },
```